tests/core/test_cells.py

```diff
@@ -90,50 +80,50 @@ def test_cell_get_cell():
     assert isinstance(hexbeam.get_cell(0), pyvista.Cell)
 
 
-@pytest.mark.parametrize("cell", cells[:5])
+@pytest.mark.parametrize("cell", cells, ids=cell_ids)
```

Comment:
Let's set variable names identically.
```suggestion
@pytest.mark.parametrize("cell", cells, cell_ids=cell_ids)
```

Reply:
I think `ids` is a kwargs of the `pytest.mark.parametrize` decorator. Did you try locally your suggestions ?

Reply:
@beroda Sorry, my mistake in checking. I withdraw my proposal.

Reply:
I checked the ["Different options for test IDs"](https://docs.pytest.org/en/7.1.x/example/parametrize.html#different-options-for-test-ids). I had mistakenly thought it was a variable that could be defined arbitrarily.